lib/ansible/plugins/doc_fragments/service_now.py

```diff
@@ -15,8 +15,18 @@ class ModuleDocFragment(object):
       - The ServiceNow instance name, without the domain, service-now.com.
       - If the value is not specified in the task, the value of environment variable C(SN_INSTANCE) will be used instead.
       - Environment variable support added in Ansible 2.9.
+      - Mutually exclusive with C(host).
       required: false
       type: str
+    host:
+      description:
+      - The ServiceNow hostname.
+      - This value is FQDN for ServiceNow host.
+      - If the value is not specified in the task, the value of environment variable C(SN_HOST) will be used instead.
```

Comment:
```suggestion
      - If the value is not specified in the task, the value of environment variable C(SN_HOST) will be used instead.
      - Environment variable support added in Ansible 2.9.
```

Reply:
it is mentioned in the ```instanse``` param description, IMO better to mention this here too

Reply:
OK.